eduNEXT/eox-nelp
# feat: supports paragon theme configuration

## Description

This allows to use paragon theme configurations or interactive color feature

## Testing instructions
1. Set interactive color
```
MFE_CONFIG.update({
    "CUSTOM_PRIMARY_COLORS": {
      "pgn-color-primary-base": "#AA0000"
    },
})
```
2. Check stats, feedback and main page
3. Set paragon configuration
```
MFE_CONFIG.update({
    "PARAGON_THEME_URLS": {
        "core": {
            "url": "https://cdn.jsdelivr.net/combine/npm/@edx/paragon@22.0.0-alpha.13/styles/css/themes/light/utility-classes.min.css,npm/@edx/paragon@22.0.0-alpha.13/dist/core.min.css"
        },
        "defaults": {
            "light": "light"
        },
        "variants": {
            "light": {
                "url": "https://css-varsify.s3.amazonaws.com/public/a9959998-0bab-4447-ada5-6819866195f3.css"
            }
        }
    },
})
```
The tenant setting is required
`"css_overrides_file_extra": "https://css-varsify.s3.amazonaws.com/public/a9959998-0bab-4447-ada5-6819866195f3.css"`

4. Check stats, feedback and main page

##With interactive color
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/2c339f84-754f-474c-a357-85a8f1784f69)
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/45660d14-e61d-4932-ad53-b5e537d4c0d3)
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/d320c167-8304-4d95-91ba-52d65ed14260)


## With paragon configuration
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/bed43d0e-7297-4f68-999b-82f8a9d96a7f)
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/f46fe4d1-3922-4e2f-ae32-603a88f81b14)
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/8ca4c82b-22d6-4788-b61c-49b00b0f1e87)


## Checklist for Merge
